src/admin/class-sign-in-with-google-admin.php

```diff
@@ -757,6 +757,49 @@ public function disconnect_account() {
 		return delete_user_meta( $current_user->ID, 'siwg_google_account' );
 	}
 
+	/**
+	 * Adds the Debug Infomation to the WordPress Site Health page.
+	 *
+	 * @hook debug_information
+	 *
+	 * @since 1.8.1
+	 * @param array $info The debug information to be added to the core information page.
+	 *
+	 * @return array $info Updated debug information to be added to the core information page.
+	 */
+	public function add_debug_info_to_site_health( $info ) {
+		$info['siwg-debug-information'] = array(
+			'label'  => __( 'Sign in with Google', 'sign-in-with-google' ),
+			'fields' => array(
+				'version'                             => array(
+					'label' => __( 'Version', 'sign-in-with-google' ),
+					'value' => $this->version,
+				),
+				'siwg_google_user_default_role'       => array(
+					'label' => __( 'Default New User Role', 'sign-in-with-google' ),
+					'value' => get_option( 'siwg_google_user_default_role', 'subscriber' ),
+				),
+				'siwg_google_domain_restriction'      => array(
+					'label' => __( 'Restrict To Domain', 'sign-in-with-google' ),
+					'value' => get_option( 'siwg_google_domain_restriction' ),
```

Comment:
Should we provide defaults for the rest of these settings? I think it would be good to see something like "none", "no", or "0" rather than blank values.

Reply:
@tarecord There is more value in knowing exactly what is in the database than setting defaults in my opinion. The only value I set a default for was the option that sets a default in the code when we get the option (role). This might open the door to a new enhancement where a class is created for getting the settings, where defaults can be set and consistently queried without having multiple places where different defaults can be set when no value exists.